software/spmd/hello_float/Makefile

```diff
@@ -1,46 +1,14 @@
-#########################################################
-# Network Configutaion
-# If not configured, Will use default Values
-#	bsg_global_X ?= $(bsg_tiles_X)
```

Comment:
Just curious - why was this Makefile changed?

Reply:
I was using it to generate vanilla_stall_trace.log. I would have used coremark or fft, if they worked, but both seem to be broken now.